devices/models/fan.py

```diff
@@ -28,6 +28,7 @@ def change_status(self):
             return 'Włączono wiatrak'

```

Comment:
Don't return hardcoded string. Return true/false values, and then map it in another method like here:

change.status() => True

```
def message(self, status):
    return f"{self.device} is turned {'on' if status else 'off'}}"

```